allennlp/nn/activations.py

```diff
@@ -55,6 +55,8 @@ def __call__(self, tensor: torch.Tensor) -> torch.Tensor:
 # Activation.by_name('relu')()
 Registrable._registry[Activation] = {
     "linear": lambda: lambda x: x,  # type: ignore
+    "mish": lambda: lambda x: x * torch.tanh(torch.nn.functional.softplus(x)), 
```

Comment:
You will need to add this `# type: ignore` statement to these lambda functions for our mypy check.

Reply:
Alright. Will make the changes.

Reply:
@DeNeutoy need a bit of help with running flake8 and black tests. Whenever I try it locally, it says module import error that no module named allennlp is found although allennlp is installed and there is no name conflicts. I fixed the code as per your review though.